docs/getting_started_gae.md

```diff
@@ -1,2 +1,51 @@
 # Getting Started for GAE
+This tutorial will give you a quick tour of GraphScope's GAE. To get started, we’ll start by installing GraphScope on local with `python3 -m pip install graphscope --upgrade`. The examples in this guide are based on Python.
 
+## Installing Environments for Compilation 
```

Comment:
Do we need this to run your code? 
Since this is a "getting started", please give SIMPLE example covered by pip installed `graphscope`.